components/director/internal/domain/api/resolver.go

```diff
@@ -81,24 +82,30 @@ func (r *Resolver) AddAPIDefinitionToPackage(ctx context.Context, packageID stri
 	}
 	defer r.transact.RollbackUnlessCommitted(tx)
 
+	log.Infof("Adding APIDefinition to package %s", packageID)
```

Comment:
you only info log that we are starting something is starting (for example to add api definition to package) and you then log when it has succeeded. Maybe you would make sense to have some more info/debug logs in the specific flows as well and not only on the surface. (for example logging what actually happens in the resolver as well as going into the service and logging any specific branching there as well.

Just to give a few examples - if im adding a package with apis right now, there will be no logs that apis are being added as they are added relatively in the service layer and also there will be no logs whether or not fetch requests are added because this happens even deeper in the service layer